projects/plugins/beta/src/class-plugin.php

```diff
@@ -394,7 +394,7 @@ public function dev_info() {
 
 		// Initialize the WP_Filesystem API.
 		require_once ABSPATH . 'wp-admin/includes/file.php';
-		$creds = request_filesystem_credentials( site_url() . '/wp-admin/', '', false, false, array() );
+		$creds = request_filesystem_credentials( site_url() . '/wp-admin/', '', false, '', array() );
```

Comment:
As above, I'd probably use one of these:
```suggestion
		$creds = request_filesystem_credentials( site_url() . '/wp-admin/', '', false, '', null );
```

```suggestion
		$creds = request_filesystem_credentials( site_url() . '/wp-admin/' );
```